devtools/package.json

```diff
@@ -4,8 +4,9 @@
 	"version": "1.0.0",
 	"private": true,
 	"description": "Preact bridge for Preact devtools",
+  "exports": "./dist/devtools.mjs",
```

Comment:
2 spaces here 🤔

Seems like the indent of `package.json` files is different, does npm no longer force some specific indent on these?

Reply:
npm now detects and uses the current indentation. I was avoiding running Prettier on these files since it converts the indentation over to tabs - we should do that in a dedicated PR probably.